store/src/main/java/com/denis/store/Store.java

```diff
@@ -2,26 +2,31 @@
 
 import com.denis.domain.Category;
 import com.denis.domain.Product;
-import com.denis.store.utility.CommandSortComparator;
 import com.denis.store.utility.RandomStorePopulator;
 
 import java.util.ArrayList;
-import java.util.Collections;
-import java.util.Comparator;
 import java.util.List;
-import java.util.stream.Collectors;
 
 public class Store {
     private List<Category> categoryList;
+    private List<Product> purchasedItems = new ArrayList<>();
+    private static Store store;
 
-    public Store() {
+    private Store() {
         try {
             this.categoryList = new RandomStorePopulator().getRandomCategory();
         } catch (Exception exception) {
             System.out.println("ERROR: RandomStorePopulator = " + exception.getMessage());
         }
     }
 
+    public static Store getInstance() {
```

Comment:
what for it is singleton? if just to apply this pattern - it is not the best choice, if not - please comment